assets/js/main.js

```diff
@@ -1,6 +1,7 @@
 $("body").on({
   mouseenter: function() {
-    if ($('.menu-main .sub-menu').is(':visible')) {
+    var submenu = $(this).find(".sub-menu");
+    if (submenu.children().length && submenu.is(':visible')) {
```

Comment:
En que escenario no se daría esta condición `submenu.is(':visible')`?

Reply:
creo se agrego porque como tiene una animacion por ahi al principio no se llega a ver y por ahi queda tosco, pero pruebo sacarselo

Reply:
Ok, si es útil dejalo, no me parece mal. Tal vez se podría poner un comentario para que no volvamos a tener la misma duda.

Reply:
ya lo saque no era necesario se ve ja